Sources/App/Core/ErrorMiddleware.swift

```diff
@@ -11,15 +11,16 @@ public final class ErrorMiddleware: Middleware {
         next.respond(to: req)
             .flatMapError { error in
                 if let abort = error as? AbortError {
-                    return self.handleError(for: req, status: abort.status)
+                    return self.handleError(for: req, status: abort.status, error: abort)
```

Comment:
Since status is a field on `error` I think we should just pass `error` and make it non-optional in `Model`.